Yank/tests/test_cli.py

```diff
@@ -109,6 +109,8 @@ def test_script_yaml():
         experiments:
             system: system
             protocol: absolute-binding
+            restraint:
+                type: FlatBottom
```

Comment:
Ah! The Boresch restraint is already included in the test you wrote for restraints, and the syntax for combinatorial restraints is tested in `test_yaml.py`, so we should be good.